app/models/verification.rb

```diff
@@ -0,0 +1,138 @@
+#  index_verifications_on_tag_id                       (tag_id)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (audio_event_id => audio_events.id) ON DELETE => cascade
+#  fk_rails_...  (creator_id => users.id)
+#  fk_rails_...  (tag_id => tags.id)
+#  fk_rails_...  (updater_id => users.id)
+#
+class Verification < ApplicationRecord
+  belongs_to :audio_event, inverse_of: :verifications
+  belongs_to :tag, inverse_of: :verifications
+  belongs_to :creator, class_name: 'User', inverse_of: :created_verifications
+  belongs_to :updater, class_name: 'User', inverse_of: :updated_verifications, optional: true
+
+  # A user can only have one verification per audio event and tag
+  validates :creator_id, uniqueness: { scope: [:audio_event_id, :tag_id] }
+
+  # Defines the possible values for confirmation
+  CONFIRMATION_TRUE = 'true'
+  CONFIRMATION_FALSE = 'false'
+  CONFIRMATION_UNSURE = 'unsure'
+  CONFIRMATION_SKIP = 'skip'
+
+  # @type [Hash{String => String}]
+  CONFIRMATION_ENUM = {
+    CONFIRMATION_TRUE => CONFIRMATION_TRUE,
+    CONFIRMATION_FALSE => CONFIRMATION_FALSE,
+    CONFIRMATION_UNSURE => CONFIRMATION_UNSURE,
+    CONFIRMATION_SKIP => CONFIRMATION_SKIP
+  }.freeze
+
+  # @!method confirmed_true?
+  #   @return [Boolean] true if the verification is confirmed as true
+  # @!method confirmed_true!
+  #   @return [void] sets the verification as confirmed true
+  # @!method confirmed_false?
+  #   @return [Boolean] true if the verification is confirmed as false
+  # @!method confirmed_false!
+  #   @return [void] sets the verification as confirmed false
+  # @!method confirmed_unsure?
+  #   @return [Boolean] true if the verification is marked as unsure
+  # @!method confirmed_unsure!
+  #   @return [void] sets the verification as unsure
+  # @!method confirmed_skip?
+  #   @return [Boolean] true if the verification is marked as skip
+  # @!method confirmed_skip!
+  #   @return [void] sets the verification as skip
+  enum :confirmed, CONFIRMATION_ENUM, prefix: :confirmed, validate: true
+
+  def self.filter_settings
+    fields = [
+      :id, :confirmed, :audio_event_id, :tag_id, :creator_id,
+      :updater_id, :created_at, :updated_at
+    ]
+
+    {
+      valid_fields: fields,
+      render_fields: fields,
+      text_fields: [:confirmed],
+      new_spec_fields: lambda { |_user|
+        {
+          confirmed: nil,
+          audio_event_id: nil,
+          tag_id: nil
+        }
+      },
+      controller: :verifications,
+      action: :filter,
+      defaults: {
+        order_by: :created_at,
+        direction: :desc
+      },
+      valid_associations: [
+        {
+          join: AudioEvent,
+          on: Verification.arel_table[:audio_event_id].eq(AudioEvent.arel_table[:id]),
+          available: true
+        },
+        {
+          join: Tag,
+          on: Verification.arel_table[:tag_id].eq(Tag.arel_table[:id]),
+          available: true
+        }
+      ]
+    }
```

Comment:
probably need to expand these - it's quite conceivable we could get a filter query for verifications like "find me all verifications which are in project 123".

But let's keep the scope of this PR small, and add the associations when @hudson-newey complains